Longest_Repeating_Subsequence/Longest_Repeating_Subsequence.cpp

```diff
@@ -0,0 +1,43 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+int LongestRepeatingSubsequence(string str){
+    int length;
+    length = str.length();
+    //Initialize all dp elements with 0
```

Comment:
A single blank line should be there before every single-line comment. Also add a space after ``//`` in every comment.